rust/src/adapter/otelstdout.rs

```diff
@@ -1,39 +1,52 @@
-use crate::{
-    adapter::otel_formater::{OtelFormatter, ResourceSpan},
-    Event, Metadata,
-};
-use std::{
-    sync::{
-        atomic::{AtomicUsize, Ordering},
-        Arc,
-    },
-    thread, time,
-};
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::Arc;
+use std::{thread, time};
+
+use crate::adapter::otel_formatter::{self, OtelFormatter, ResourceSpan, Span};
+use crate::adapter::{Adapter, Collector};
+use crate::{add_to_linker, Event, Metadata};
```

Comment:
Did you use a tool to reformat these? The lsp added them in the nested format and rustfmt & clippy were happy with it so I didn't change it. This looks better than before, but I'm not sure what the preferred format is.

Reply:
I don't have a tool that formats these, but I generally shy away from nesting the dependencies quite so deep. 

I like to keep them on a single line, and limit the scope to a single module (just std::time, not std::{time, sync} etc). 

Sometimes the auto-importers in LSPs go against this, so it's just the matter of a light pass over the code to clean it up.